src/GameQ/Protocols/Gta5m.php

```diff
@@ -163,6 +185,15 @@ protected function processStatus(Buffer $buffer)
             $result->add($key, $val);
         }
 
+        if ($this->PlayerList) {
+            $players = [];
+            $data = json_decode($this->PlayerList);
+            foreach ($data as $player) {
+                $players[] = array("id"=>$player->id,"name"=>$player->name,"ping"=>$player->ping,"identifiers"=>$player->identifiers);
+            }
+            $result->add('players', $players);
+        }
+
         /*var_dump($data);
```

Comment:
Remove commented code used when building/testing.

Reply:
you mean this? 
/*var_dump($data);